app/assets/stylesheets/application/safecast.scss

```diff
@@ -466,7 +488,7 @@ ul.measurement-listed {
 }
 
 .bootstrap-datetimepicker-widget {
-  z-index: 10000;
+  z-index: 10000 !important;
```

Comment:
Can you provide a little more info on why we needed important here and on `#mobile-navbar`?

Reply:
Leftovers from trying to solve an overlapping issue with the datetime pickers. Will take them out.